config/default.json

```diff
@@ -31,5 +31,12 @@
   "matchingQueue": {
     "numberOfWorkers": 2,
     "pollingInterval": 1000
+  },
+  "validation": {
+    "enabled": false,
+    "fhirVersion": "dstu2",
```

Comment:
Lets move this config property out of this validation grouping, as the fhirVersion will be used by other features as well, like the XML/JS conversions